test/e2e/qat/qatplugin_kernel.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package qat
+
+import (
+	"time"
+
+	"github.com/intel/intel-device-plugins-for-kubernetes/test/e2e/utils"
+	"github.com/onsi/ginkgo"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+)
+
+const (
+	qatPluginKernelYaml = "deployments/qat_plugin/base/intel-qat-kernel-plugin.yaml"
+)
+
+func init() {
+	ginkgo.Describe("QAT plugin in kernel mode", describeQatKernelPlugin)
+}
+
+func describeQatKernelPlugin() {
+	f := framework.NewDefaultFramework("qatpluginkernel")
```

Comment:
I believe this is also run by default? How to make it 'opt-in' to be able to run this suite also on machines without the out-of-tree driver?

Reply:
Right. I've added a feature-gate check which disables the test case if `-feature-gates enable-experimental=true` option is not provided

Reply:
+1